src/sentry/shared_integrations/client/base.py

```diff
@@ -129,13 +143,25 @@ def _request(
         )
         _prepared_request = prepared_request if prepared_request is not None else request.prepare()
 
-        with sentry_sdk.start_transaction(
-            op=f"{self.integration_type}.http",
-            name=f"{self.integration_type}.http_response.{self.name}",
-            parent_span_id=parent_span_id,
-            trace_id=trace_id,
-            sampled=random() < 0.05,
+        with (
+            sentry_sdk.start_transaction(
+                op=f"{self.integration_type}.http",
+                name=f"{self.integration_type}.http_response.{self.name}",
+                parent_span_id=parent_span_id,
+                trace_id=trace_id,
+                sampled=random() < 0.05,
```

Comment:
We could probably remove this and let dynamic sampling do it's thing. What do you think?

Reply:
There are plenty of other transactions where we manually set the sample rate, it's just that we do it [in the `traces_sampler`](https://github.com/getsentry/sentry/blob/d062e57d097b2212c8e04169757b176ba13ac1b5/src/sentry/utils/sdk.py#L244-L266), based on either incoming endpoint hit or task name. Since this doesn't fall into one of those categories, it's gotta be done where the transaction is started. 

In principle I'm not against changing the sampling rate, but I think if we do it should be done separately, because a) general separation of concerns, and b) IDK what `settings.SENTRY_BACKEND_APM_SAMPLING ` is set to for SAAS, but the default value is `0`, which means that removing the `sampled=` line here would prevent the transaction from ever being sent.

Reply:
wfm